module/zfs/vdev_queue.c

```diff
@@ -913,6 +918,10 @@ vdev_queue_last_offset(vdev_t *vd)
 module_param(zfs_vdev_aggregation_limit, int, 0644);
 MODULE_PARM_DESC(zfs_vdev_aggregation_limit, "Max vdev I/O aggregation size");
 
+module_param(zfs_vdev_aggregation_limit_non_rotating, int, 0644);
+MODULE_PARM_DESC(zfs_vdev_aggregation_limit_non_rotating,
+	"Max vdev I/O aggregation size for non-rotating media");
```

Comment:
Can you add an entry for this in `man/man5/zfs-module-parameters.5`.

Reply:
Added.